Nodejs/MigrateToJsps/Jsps/EsprojFileModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using System.Xml.Serialization;
+
+namespace MigrateToJsps
+{
+    [XmlRoot(ElementName = "Project")]
+    public class EsprojFile
+    {
+        [XmlAttribute(AttributeName = "Sdk")]
+        public string Sdk = @"Microsoft.VisualStudio.JavaScript.Sdk/0.5.64-alpha";
```

Comment:
Latest is now 0.5.85-alpha: https://www.nuget.org/packages/Microsoft.VisualStudio.JavaScript.SDK/0.5.85-alpha

Reply:
done!